spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/R2dbcTransactionManager.java

```diff
@@ -373,7 +373,19 @@ protected Mono<Void> doCleanupAfterCompletion(TransactionSynchronizationManager
 					txObject.getConnectionHolder().clear();
 				}
 				return Mono.empty();
-			}));
+			})).onErrorResume(e -> {
```

Comment:
I agree with you.
It seems like setIsolationLevel has been removed since fae36e98b4c3d72941e3f209a87047332ff51ba1.
Here is the updated code: 3724bb0
Thank you!